x/proof/types/params.go

```diff
@@ -1,30 +1,34 @@
 package types
 
 import (
+	"encoding/hex"
+
 	"cosmossdk.io/math"
 	cosmostypes "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 
 	"github.com/pokt-network/poktroll/app/volatile"
 	"github.com/pokt-network/poktroll/pkg/client"
+	"github.com/pokt-network/poktroll/pkg/crypto/protocol"
 )
 
 var (
 	_ client.ProofParams  = (*Params)(nil)
 	_ paramtypes.ParamSet = (*Params)(nil)
 
-	KeyMinRelayDifficultyBits                = []byte("MinRelayDifficultyBits")
-	ParamMinRelayDifficultyBits              = "min_relay_difficulty_bits"
-	DefaultMinRelayDifficultyBits    uint64  = 0 // TODO_MAINNET(#142, #401): Determine the default value.
-	KeyProofRequestProbability               = []byte("ProofRequestProbability")
-	ParamProofRequestProbability             = "proof_request_probability"
-	DefaultProofRequestProbability   float32 = 0.25 // See: https://github.com/pokt-network/pocket-core/blob/staging/docs/proposals/probabilistic_proofs.md
-	KeyProofRequirementThreshold             = []byte("ProofRequirementThreshold")
-	ParamProofRequirementThreshold           = "proof_requirement_threshold"
-	DefaultProofRequirementThreshold uint64  = 20 // See: https://github.com/pokt-network/pocket-core/blob/staging/docs/proposals/probabilistic_proofs.md
-	KeyProofMissingPenalty                   = []byte("ProofMissingPenalty")
-	ParamProofMissingPenalty                 = "proof_missing_penalty"
-	DefaultProofMissingPenalty               = cosmostypes.NewCoin(volatile.DenomuPOKT, math.NewInt(320)) // See: https://github.com/pokt-network/pocket-core/blob/staging/docs/proposals/probabilistic_proofs.md
+	KeyMinRelayDifficultyBits                   = []byte("MinRelayDifficultyBits")
+	ParamRelayDifficultyTargetHash              = "relay_difficulty_target_hash"
+	DefaultRelayDifficultyTargetHashHex         = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff" // all relays are payable
```

Comment:
Should `DefaultRelayDifficultyTargetHashHex` be an attribute of the `Service` type?

Reply:
No. One of the followup items I have in #699 is to remove the parameter and make default be a global min.

If any service has 1 relay per session, difficulty should be minimum.